lib/DocumentsFieldArray/FileUploaderField.js

```diff
@@ -74,18 +86,22 @@ class FileUploaderField extends React.Component {
 
   render() {
     return (
-      <FileUploaderFieldView
-        error={this.props.meta.error || this.state.error}
-        file={this.props.input.value ? this.state.file : {}}
-        isDropZoneActive={this.state.isDropZoneActive}
-        onDelete={this.handleDelete}
-        onDownloadFile={this.props.onDownloadFile}
-        onDragEnter={() => this.setState({ isDropZoneActive: true })}
-        onDragLeave={() => this.setState({ isDropZoneActive: false })}
-        onDrop={this.handleDrop}
-        uploadInProgress={this.state.uploadInProgress}
-        {...pickBy(this.props, (_, key) => key.startsWith('data-test-'))}
-      />
+      <IntlConsumer>
```

Comment:
Woof. This looks _rough_, but I don't really see a better way of writing it. Seems like the kind of thing we should refactor to use `injectIntl` when we get onto `react-intl` v3.0. Can you put a comment above this `return` saying that so we don't forget? Something like:

```/* TODO: Refactor this component to use `injectIntl` when Folio starts using react-intl 3.0 */```